shardingsphere-scaling/shardingsphere-scaling-bootstrap/src/main/resources/conf/server.yaml

```diff
@@ -19,3 +19,8 @@ port: 8888
 blockQueueSize: 10000
 pushTimeout: 1000
 workerThread: 30
+#name: elasticjob
+#registryCenter:
+#  type: zookeeper
+#  serverLists: localhost:2181
+#  props:
```

Comment:
Need leave a blank line in the end of file

Reply:
ok